test/models/hold_test.rb

```diff
@@ -89,6 +89,20 @@ class HoldTest < ActiveSupport::TestCase
     assert_equal change.changeset["case_number"], [nil, "2"]
   end
 
+  test 'can list the create dates and names of the parent thesis files' do
+    h = holds(:valid)
+    f = Rails.root.join('test','fixtures','files','a_pdf.pdf')
+    h.thesis.files.attach(io: File.open(f), filename: 'a_pdf.pdf')
+    thesis_file = h.thesis.files.first
+    assert_equal "a_pdf.pdf", thesis_file.filename.to_s
+    assert_equal h.dates_thesis_files_received, "#{thesis_file.created_at.strftime('%Y-%m-%d')} (#{thesis_file.filename.to_s})"
```

Comment:
I feel like this is reimplementing the model method in the test and checking it's the same output.

While less convenient, it would probably be better to either explicitly construct what we expect here or look for a pattern that doesn't require reimplementing the method under test.